packages/amplify-codegen-appsync-model-plugin/src/utils/process-auth.ts

```diff
@@ -92,31 +92,4 @@ export function processAuthDirective(directives: CodeGenDirectives): AuthDirecti
       },
     };
   });
-}
-
-export function getOwnerAuthRules(modelObj: CodeGenModel): AuthRule[] {
```

Comment:
I'm not familiar with this code, but it seems like we are removing `getOwnerAuthRules` (and getOwnerFieldName, etc..) from everywhere.  Can anyone verify that this is the right thing to do?